### PR TITLE
Misc updates and fixes

### PR DESCRIPTION
**Pull request by gwillen**
- add requirements.txt listing package dependencies
- fix projector window width calculation bug
- enable double-clicking in projector window to fullscreen / un-fullscreen it (and remove titlebar offset)

### CODE REVIEW

File: projector.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ DEST_CORNERS = [
   [0, 768], # bottom left
 ]
 
-VERTICAL_OFFSET = -45 # For title bar
+VERTICAL_OFFSET = 0 # No longer needed with fullscreen feature
```
Comment (saoirse-zee) [inline projector.toml]: I'm wondering if we can get rid of [the vertical transformation](https://github.com/saoirse-zee/tinyland/blob/82cad20d336250cf4c14353b168f783b91b7bff7/tinyland.py#L17) altogether. Is there a reason we need to keep it?

Comment (gwillen) [reply projector.toml]: Yeah, that's true! I will update the pull request, hold on.

**Comment by gwillen** [comment]
Ok, fixed! Apologies for the extra noise in the diffs; my editor is configured to automatically remove whitespace from the ends of lines. (Possibly you could configure VS Code to do the same.)

**Comment by gwillen** [comment]
Yay! 🎉

**Pull request merged**